pkg/reconciler/pipelinerun/pipelinerun_expired.go

```diff
@@ -0,0 +1,174 @@
+package pipelinerun
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/controller"
+	"knative.dev/pkg/apis"
+
+	apispipeline "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+)
+
+func (tc *Reconciler) AddPipelineRun(obj interface{}) {
+	pr := obj.(*apispipeline.PipelineRun)
+	klog.V(4).Infof("Adding PipelineRun %s/%s", pr.Namespace, pr.Name)
+
+	if pr.DeletionTimestamp == nil && pipelineRunCleanup(pr) {
+		tc.PrEnqueue(pr)
+	}
+}
+
+func (tc *Reconciler) UpdatePipelineRun(old, cur interface{}) {
+	pr := cur.(*apispipeline.PipelineRun)
+	klog.V(4).Infof("Updating PipelineRun %s/%s", pr.Namespace, pr.Name)
```

Comment:
the reconciler object actually has a `Logger` object which we usually use for logging, e.g.

https://github.com/tektoncd/pipeline/blob/192134488deec8d58a9ee4c339c797577f933734/pkg/reconciler/taskrun/taskrun.go#L100

Reply:
Thx, I'll fix it